sql/npc_list.sql

```diff
@@ -15231,6 +15232,7 @@ INSERT INTO `npc_list` VALUES (17412795,'blank','     ',0,0,0,0,0,50,50,0,0,0,2,
 INSERT INTO `npc_list` VALUES (17412796,'csnpc','noname',0,0,0,0,0,50,50,0,0,0,6,27,0x0100000100100020003000400050006000700000,32,'WOTG');
 INSERT INTO `npc_list` VALUES (17412797,'debugNPC','Debug NPC',192,-6.5,0,-22.5,1,50,50,0,0,0,2,27,0x0000bd0800000000000000000000000000000000,32,'WOTG');
 INSERT INTO `npc_list` VALUES (17412798,'debugNPC','Debug NPC',128,-370,-52,-112,1,50,50,0,0,0,2,27,0x0000bd0800000000000000000000000000000000,32,'WOTG');
+INSERT INTO `npc_list` VALUES (17441098,'HomePoint#1','Home Point #1',128,-554,-70,66,1,40,40,0,0,0,0,3,0x0000330000000000000000000000000000000000,0,'WOTG');
```

Comment:
your welcome.
yea i remember vegly from about 2 + yrs ago when i did some changes that was one of the things was staying consistent. and so i didn't just drop my db i edited the file by hand for consistency.